src/OrderLinesList/Details/OrderLineDetails.js

```diff
@@ -1,32 +1,35 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import ReactRouterPropTypes from 'react-router-prop-types';
 import { get } from 'lodash';
 
+import {
+  baseManifest,
+  Tags,
+} from '@folio/stripes-acq-components';
+
 import { POLineView } from '../../components/POLine';
 import {
   ORDERS_API,
   LINES_API,
 } from '../../components/Utils/api';
+
 import { FILTERS as ORDER_FILTERS } from '../../OrdersList';
 
 class OrderLineDetails extends Component {
   static manifest = Object.freeze({
     order: {
-      type: 'okapi',
-      path: `${ORDERS_API}/%{orderId}`,
-      throwErrors: false,
-      fetch: false,
-      accumulate: true,
+      ...baseManifest,
+      path: (queryParams, pathComponents, resourceData, logger, props) => {
+        const orderId = get(props, ['resources', 'orderLine', 'records', 0, 'purchaseOrderId']);
+
+        return `${ORDERS_API}/${orderId}`;
```

Comment:
`orderId` might be null or undefined, right?

Reply:
Yep. Fixed